src/components/Chakra/Box.stories.mdx

```diff
@@ -17,7 +17,8 @@ import DSProvider from "../../theme/provider";
 | ----------------- | ---------- |
 | Added             | `0.24.0`   |
 
-Note: This needs the use of the `DSProvider` component. See the [README](https://github.com/NYPL/nypl-design-system#using-the-design-system-in-your-product)
+Note: This needs the use of the `DSProvider` component. See the
+[README](https://github.com/NYPL/nypl-design-system#using-the-design-system-in-your-product)
```

Comment:
Why link to the readme instead of to the `DSProvider` section of storybook https://pr772-vtdmwsvngru6lcxs6iqzvha0nldmtabw.tugboat.qa/?path=/docs/chakra-ui--page#dsprovider ?  (Just think this is nice in terms of not causing the user switch between apps)

Reply:
Good point.